library/src/main/java/com/bumptech/glide/load/model/Model.java

```diff
@@ -0,0 +1,28 @@
+package com.bumptech.glide.load.model;
+
+/**
+ * An optional interface that models can implement to enhance control over Glide behaviors.
+ */
+public interface Model {
+
+    /**
+     * Returns {@code true} if this model produces the same image using the same mechanism
+     * (server, authentication, source etc) as the given model.
+     * <p/>
```

Comment:
<p>

Reply:
@sjudd There's actually no text with your comment so no idea what you want :)

Reply:
haha my bad, ``<p>``. I forgot that would just get converted into whitespace :)